auth_proxy/static/main.js

```diff
@@ -0,0 +1,22 @@
+$(function () {
+  $('[rel=post]').on('click', function (event) {
+    event.preventDefault();
+
+    var $el = $(event.currentTarget),
+        $form = $('<form />', {
+          action: $el.attr('href'),
+          method: 'post'
+        });
+
+    $.each($el.data(), function (key, value) {
+      $form.append($('<input />', {
+        name: key,
+        value: value
+      }));
+    });
+
+    if (confirm('Are you sure?')) {
```

Comment:
This confirm should be replaced with a dialog like https://docs.google.com/presentation/d/1kQU0_dDL2vNdAuI4iUatthSTo7BxV_Hrtx2LuO4D6p8/edit#slide=id.p43

![image](https://cloud.githubusercontent.com/assets/313089/16964645/63605686-4db1-11e6-89c3-ad3f0587d3b4.png)

Happy to merge first and do that later if you prefer; let me know.